entry/src/main/resources/base/layout/ability_main.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<StackLayout xmlns:ohos="http://schemas.huawei.com/res/ohos"
+             xmlns:app="http://schemas.huawei.com/res-auto"
+             ohos:width="match_parent"
+             ohos:height="match_parent"
+             ohos:orientation="horizontal">
+
+    <per.goweii.reveallayout.RevealLayout
```

Comment:
Size of the components to be reduced and use english

Reply:
Review comment reworked and kept same as orginal lib